pkg/provisioning/keypair.go

```diff
@@ -0,0 +1,133 @@
+package provisioning
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh"
+)
+
+const (
+	privateKeyFile = "serviceaccount-signer.private"
+	publicKeyFile  = "serviceaccount-signer.public"
+)
+
+func createKeys(prefixDir string) {
+
+	privateKeyFilePath := filepath.Join(prefixDir, privateKeyFile)
+	publicKeyFilePath := filepath.Join(prefixDir, publicKeyFile)
+	bitSize := 4096
+
+	_, err := os.Stat(privateKeyFilePath)
+	if err == nil {
+		log.Printf("Using existing RSA keypair found at %s", privateKeyFilePath)
+		return
+	}
+
+	log.Print("Generating RSA keypair")
+	privateKey, err := rsa.GenerateKey(rand.Reader, bitSize)
+	if err != nil {
+		log.Fatal(err.Error())
```

Comment:
```suggestion
log.WithError(err).Fatal("Failed to generate private key")
```

Just following the way we do it in hive.